index.bs

```diff
@@ -109,8 +133,9 @@ The GeolocationSensor Interface {#geolocationsensor-interface}
 -------------------------------
 
 <pre class="idl">
-  [Constructor(optional SensorOptions options), SecureContext, Exposed=Window]
+  [Constructor(optional GeolocationSensorOptions options), SecureContext, Exposed=Window]
   interface GeolocationSensor : Sensor {
+    static Promise&lt;GeolocationSensorReading&gt; read(optional ReadOptions readOptions);
```

Comment:
Also, wouldn't it be better to call this method `readOnce`? 
IMO it makes more clear the difference between this method and the `onreading` event handler from the user perspective.